effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -580,19 +580,22 @@ object Transformer {
   }
 
   def pushFrameOnto(stack: Operand, environment: machine.Environment, returnAddressName: String, sharer: Operand, eraser: Operand)(using ModuleContext, FunctionContext, BlockContext) = {
+    if (!environment.isEmpty) {
```

Comment:
Should we create an additional function for this? (like in `popEnvironmentFrom`)